pkg/slo-controller/nodeslo/nodeslo_controller.go

```diff
@@ -207,14 +179,13 @@ func (r *NodeSLOReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 }
 
 func (r *NodeSLOReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	configMapCacheHandler := NewSLOCfgHandlerForConfigMapEvent(r.Client, DefaultSLOCfg())
+	r.sloCfgCache = &configMapCacheHandler.SLOCfgCache
```

Comment:
`handler.GetCache()`

Reply:
nodeslo handler is in same package, so cache can be accessed in controller directly even we change SLOCfgCache as sloCfgCache and make a GetCache function